Modules/Filtering/FastMarching/include/itkFastMarchingUpwindGradientImageFilter.h

```diff
@@ -23,6 +23,33 @@
 
 namespace itk
 {
+
+/** \class FastMarchingUpwindGradientImageFilterEnums
+ *
+ * \brief enums for itk::FastMarchingUpwindGradientImageFilter
+ *
+ * \ingroup ITKFastMarching
+ */
+class FastMarchingUpwindGradientImageFilterEnums
+{
+public:
+  /** \class TargetCondition
+   * \ingroup ITKFastMarching
+   * Specify how many targets are necessary to determine when the front must stop.
+   */
+  enum class TargetCondition : int
```

Comment:
Did not put `uint8_t` as the former instance variable was an `int`: https://github.com/InsightSoftwareConsortium/ITK/pull/3863/files#diff-7309891cf2dfb339749626e3d6a7bfcc155bcbbacee8b8b70cbaa665d518aad1L291, and I considered that the API should be kept. Let me know if we think otherwise.